pentaho-requirejs-compressor/pom.xml

```diff
@@ -17,16 +17,19 @@
     <dependency>
       <groupId>com.fasterxml.jackson.core</groupId>
       <artifactId>jackson-core</artifactId>
+      <version>${fasterxml-jackson.non-osgi.version}</version>
```

Comment:
test code, not really needed, unless there's weird compile issue. The major version 2.X coming from the parent-maven-poms should suffice.